integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/ConfigMap.java

```diff
@@ -31,4 +35,31 @@ public static boolean delete(String name, String namespace) {
     return Kubernetes.deleteConfigMap(name, namespace);
   }
 
+  /**
+   * Check whether the configmap exists.
+   * @param name name of the config map
+   * @param namespace namespace where config map exists
+   * @return true if the config map exists, false otherwise
+   * @throws ApiException if Kubernetes client API call fails
+   */
+  public static boolean doesCMExist(String name, String namespace) throws ApiException {
+    List<V1ConfigMap> configMaps = new ArrayList<>();
+    V1ConfigMapList configMapList = Kubernetes.listConfigMaps(namespace);
+
+    if (configMapList != null) {
+      configMaps = configMapList.getItems();
+    }
+
+    for (V1ConfigMap configMap : configMaps) {
+      if (configMap.getMetadata() != null) {
```

Comment:
Combine these three if's into one with &&.

Reply:
fixed

Reply:
I think what Ryan was suggestion was:

```
    for (V1ConfigMap configMap : configMaps) {
      if (configMap.getMetadata() != null &&
          configMap.getMetadata().getName() != null &&
          configMap.getMetadata().getName().equals(name)) {
            return true;
      }
    }

Reply:
yes, this is already changed in the latest commit.